pyroma/ratings.py

```diff
@@ -248,6 +248,25 @@ class License(FieldTest):
     weight = 50
     field = 'license'
 
+    def test(self, data):
+        self._license_specified = super(License, self).test(data)
+        if not self._license_specified:
+            return False
+
+        classifiers = data.get('classifiers', [])
+        for classifier in classifiers:
+            parts = [p.strip() for p in classifier.split('::')]
+            if parts[0] == 'License':
+                # license classifier exist
+                self.weight = 100
```

Comment:
I'd rather see that the weight is 20 if there is a license field, but not a classifier.
Having the classifier is IMO less important than having the licence field.

Otherwise this is good.